tests/ops/op_math/test_pow_op.py

```diff
@@ -476,7 +516,7 @@ def test_depth_property(self):
 
     def test_simplify_nested_pow_ops(self):
         """Test the simplify method with nested pow operations."""
-        pow_op = Pow(base=Pow(base=qml.adjoint(Pow(base=qml.CNOT([1, 0]), z=1.2)), z=2), z=5)
+        pow_op = Pow(base=Pow(base=qml.adjoint(Pow(base=qml.CNOT([1, 0]), z=2)), z=1.2), z=5)
```

Comment:
This essentially does not change what's being tested, and it avoids the (newly) unsupported adjoint of a fractional power.

Reply:
Nice!